src/jit/lsra.cpp

```diff
@@ -10173,167 +10117,140 @@ void LinearScan::TupleStyleDump(LsraTupleDumpMode mode)
                    splitEdgeInfo.toBBNum);
         }
 
-        for (GenTree* statement = block->FirstNonPhiDef(); statement; statement = statement->gtNext)
+        for (GenTree* node : LIR::AsRange(block).NonPhiNodes())
         {
-            if ((statement->gtFlags & GTF_STMT_TOP_LEVEL) == 0)
+            // TODO(pdg): fix the dump below for LIR
```

Comment:
I'm not sure what the current issue is with the Tuple Style dump, but I find it much less useful than the table-style dump, and the dump of the raw RefPositions.